lib/events/eventstest/mock_emitter.go

```diff
@@ -30,6 +30,14 @@ type MockEmitter struct {
 	events []events.AuditEvent
 }
 
+func (e *MockEmitter) Write(_ []byte) (int, error) {
```

Comment:
```suggestion
func (e *MockEmitter) Write(b []byte) (int, error) {
```

Or `_` out `event` below. Just be consistent.